docs/section5/Rule5-19.md

```diff
@@ -18,34 +18,32 @@
 **Data Lookup:** Table G3.1.1-1  
 **Function Call:**  
 
-  1. get_building_segment_window_wall_areas()  
+  1. get_area_type_window_wall_areas()  
   2. data_lookup()  
   3. match_data_element()
 
 ## Rule Logic:  
 
-- Get window wall areas dictionary for B_RMR: `window_wall_areas_dictionary_b = get_building_segment_window_wall_areas(B_RMR)`
+- Get window wall areas dictionary for B_RMR: `window_wall_areas_dictionary_b = get_area_type_window_wall_areas(B_RMR)`
 
-- Get window wall areas dictionary for P_RMR: `window_wall_areas_dictionary_p = get_building_segment_window_wall_areas(P_RMR)`
+- Get window wall areas dictionary for P_RMR: `window_wall_areas_dictionary_p = get_area_type_window_wall_areas(P_RMR)`
 
 - For each building segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`
 
-  - Check if building segment area type is not included in Table G3.1.1-1: `if NOT data_lookup(table_G3_1_1_1, building_segment_b.area_type_vertical_fenestration):`
+  - Check if building segment area type is not included in Table G3.1.1-1: `if NOT building_segment_b.area_type_vertical_fenestration:`
 
-    - Calculate building segment window wall ratio: `building_segment_wwr_b = window_wall_areas_dictionary_b[building_segment_b][0] / window_wall_areas_dictionary_b[building_segment_b][1]`
+    - Calculate building segment window wall ratio: `area_type_wwr_b = window_wall_areas_dictionary_b["NONE"]["TOTAL_WINDOW_AREA"] / window_wall_areas_dictionary_b["NONE"]["TOTAL_WALL_AREA"]`
 
-    - Get matching building segment in P_RMR: `building_segment_p = match_data_element(P_RMR, BuildingSegments, building_segment_b.id)`
-
-      - Calculate building segment window wall ratio in P_RMR: `building_segment_wwr_p = window_wall_areas_dictionary_p[building_segment_p][0] / window_wall_areas_dictionary_p[building_segment_p][1]`
+    - Calculate building segment window wall ratio in P_RMR: `area_type_wwr_p = window_wall_areas_dictionary_p["NONE"]["TOTAL_WINDOW_AREA"] / window_wall_areas_dictionary_p["NONE"]["TOTAL_WALL_AREA"]`
 
       **Rule Assertion:**
 
-      - Case 1: If building is all new and building segment window-wall-ratio is equal to that in P_RMR or 40%, whichever is smaller: `if ( B_RMR.building.is_all_new ) AND ( building_segment_wwr_b == min(building_segment_wwr_p, 0.4) ): PASS`
+      - Case 1: If building is all new and window-wall-ratio for area type not in Table G3.1.1-1 is equal to that in P_RMR or 40%, whichever is smaller: `if ( B_RMR.building.is_all_new ) AND ( area_type_wwr_b == min(area_type_wwr_p, 0.4) ): PASS`
 
```

Comment:
This should be confirming that all relevant building_segments are is_all_new. This applies to other "cases" as well. Also need to update the TCDs.